z3c/dependencychecker/__init__.py

```diff
@@ -1 +1,4 @@
 # -*- coding: utf-8 -*-
+import logging
+
+logger = logging.getLogger('dependencychecker')
```

Comment:
I dislike having stuff in an `__init__` unless absolutely necessary.

Reply:
@reinout fair point, I put it there as a way to avoid having to create a 2 lines module for it, another solution that I don't like is adding it to main.py or any other random module, as then you keep importing that module from everywhere just for the logger.

I'm fine with creating a logger.py if you think is best.

Reply:
You don't need a `logger.py`! You can just have a `logger = logging.getLogger(__name__)` line in every file. That's the default in all my projects (and also in django, I believe).